edb/pgsql/common.py

```diff
@@ -121,8 +123,11 @@ def quote_type(type_):
     return first + last
 
 
-def get_module_backend_name(module, prefix='edgedb_'):
-    return edgedb_name_to_pg_name(f'{prefix}{module}', len(prefix))
+def get_module_backend_name(module):
+    # standard modules go into "edgedbstd", user ones into "edgedbpub"
+    assert isinstance(module, str) and "-" not in module, (
```

Comment:
Do we have uuid cast to strings floating around?

Reply:
Actually, a dash in the module name isn't prohibited, I can do 

```
CREATE MODULE `foo-bar`
```

so this will trip up.  I suggest dropping the check.

Reply:
@msullivan Sully, could you please add one test that uses a quoted module name with a space in it?